src/platform/packages/private/kbn-esql-editor/src/types.ts

```diff
@@ -122,6 +123,7 @@ export interface EsqlPluginStartBase {
   variablesService: ESQLVariableService;
   getLicense: () => Promise<ILicense | undefined>;
   getInferenceEndpointsAutocomplete: () => Promise<InferenceEndpointsAutocompleteResult>;
+  buildFlavor: BuildFlavor;
```

Comment:
Just an isServerless boolean here. This builFlavor is very core thingy, we don't care about anything else than serverless